hamcrest/Hamcrest/File/FileMatcher.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Please remove this and don't use non-PHP 5.3+ (from `composer.json`) incompatible code.

---

I haven't seen this in any of the other files.

Reply:
Ok, I saw that the tests are running for PHP 7.0 and above, so I assumed that the first supported version is PHP 7.0. Will change the code to be compatible with PHP 5.3